week-1/articles/installfest.md

```diff
@@ -39,12 +51,7 @@ You will also be show how to sign up for a user account on the following service
 2. Click on the link for Windows
 3. Run the installer from your downloads folder
```

Comment:
Advise of the version they need to use? Stable vs LTS causes issues with proxies when starting React and I'm sure some other issues [only one I know of in the curriculum currently].